src/bo4e/bo/lastgang.py

```diff
@@ -0,0 +1,129 @@
+"""
+Contains LastgangKompakt class
+and corresponding marshmallow schema for de-/serialization
+"""
+from typing import List, Optional
+
+import attr
+from marshmallow import fields
+from marshmallow_enum import EnumField  # type:ignore[import]
+
+from bo4e.bo.geschaeftsobjekt import Geschaeftsobjekt, GeschaeftsobjektSchema
+from bo4e.com.tagesvektor import Tagesvektor, TagesvektorSchema
+from bo4e.com.zeitintervall import Zeitintervall, ZeitintervallSchema
+from bo4e.com.zeitreihenwert import Zeitreihenwert, ZeitreihenwertSchema
+from bo4e.enum.botyp import BoTyp
+from bo4e.enum.lokationstyp import Lokationstyp
+from bo4e.enum.mengeneinheit import Mengeneinheit
+from bo4e.enum.sparte import Sparte
+from bo4e.validators import check_list_length_at_least_one, obis_validator
+
+
+# pylint: disable=too-many-instance-attributes, too-few-public-methods
+@attr.s(auto_attribs=True, kw_only=True)
+class _LastgangBody:
+    """
+    The LastgangBody is a mixin that contains the "body" of a Lastgang that is used in both the :class:`Lastgang` as
+    well as :class:`LastgangKompakt`.
+    """
+
+    #: Angabe, ob es sich um einen Gas- oder Stromlastgang handelt
+    sparte: Sparte = attr.ib(validator=attr.validators.instance_of(Sparte))
+
+    #: Eindeutige Nummer der Marktlokation bzw der Messlokation, zu der der Lastgang gehört
+    lokations_id: str = attr.ib(validator=attr.validators.instance_of(str))
+
+    #: Marktlokation oder Messlokation
+    lokationstyp: str = attr.ib(validator=attr.validators.instance_of(Lokationstyp))
+    # todo: implement a lokations-id + lokationstyp cross check (such that lokationstyp malo checks for valid malo id)
+
+    #: Definition der gemessenen Größe anhand ihrer Einheit
+    messgroesse: Mengeneinheit = attr.ib(validator=attr.validators.instance_of(Mengeneinheit))
+
+    # optional attributes
+    #: Versionsnummer des Lastgangs
+    version: Optional[str] = attr.ib(default=None, validator=attr.validators.optional(attr.validators.instance_of(str)))
+    #: Die OBIS-Kennzahl für den Wert, die festlegt, welche Größe mit dem Stand gemeldet wird, z.B. '1-0:1.8.1'
```

Comment:
Woher kommt denn dieser Docstring? :D
In der Doku hab ich diesen gefunden:
```suggestion
    #: Genormte OBIS-Kennzahl zur Kennzeichnung der Messgröße
```

Reply:
das hatte ich glaube ich aus dem anderen lastgang kopiert.

Reply:
Hab ich extra geschaut, das ist bei beiden Lastgängen der Kommentar. Aber vllt. einfach von woanders wo eine OBIS-Kennzahl genutzt wird.